test/spec/routes_spec.rb

```diff
@@ -0,0 +1,219 @@
+    @driver.navigate.to "#{@config['self_url']}/#{data[:team].slug}/suggested-matches/#{query}"
+
+    wait_for_selector('.search__list-header-filter-row')
+    expect(@driver.page_source.include?('This page does not exist or you do not have authorized access.')).to be(false)
+    expect(@driver.find_elements(:css, '.list-sort').empty?).to be(false)
+  end
+
+  it 'should load route :team/assigned-to-me(/:query)', bin3: true do
+    data = api_create_team
+    query = '%7B%22archived%22%3A4%2C%22sort%22%3A%22recent_activity%22%2C%22sort_type%22%3A%22ASC%22%2C%22parent%22%3A%7B%22type%22%3A%22team%22%2C%22slug%22%3A%22fsaf%22%7D%7D'
+    @driver.navigate.to "#{@config['self_url']}/#{data.slug}/assigned-to-me/#{query}"
+
+    wait_for_selector('.search__list-header-filter-row')
+    expect(@driver.page_source.include?('This page does not exist or you do not have authorized access.')).to be(false)
+    expect(@driver.find_elements(:css, '.list-sort').empty?).to be(false)
+  end
+
+  it 'should load route :team/bot', bin2: true do
+    data = api_create_team(is_admin: true)
+    @driver.navigate.to "#{@config['self_url']}/#{data.slug}/bot"
+    wait_for_selector('.test__alert-content')
+    expect(@driver.page_source.include?('This page does not exist or you do not have authorized access.')).to be(false)
+    expect(@driver.find_elements(:css, '.contactId').empty?).to be(false)
+  end
+
+  it 'should load route :team/feed/{tab}', bin3: true do
+    data = api_create_feed_with_item
+
+    @driver.navigate.to "#{@config['self_url']}/#{data[:team].slug}/feed/#{data[:feed].feed['id']}/edit"
+    wait_for_selector('#create-feed__description')
+    expect(@driver.page_source.include?('This page does not exist or you do not have authorized access.')).to be(false)
+    expect(@driver.find_elements(:css, '#create-feed__title').empty?).to be(false)
+
+    @driver.navigate.to "#{@config['self_url']}/#{data[:team].slug}/feed/#{data[:feed].feed['id']}/item/#{data[:feed].cluster['project_media_id']}"
+    wait_for_selector('#feed-item-page-teams')
+    expect(@driver.page_source.include?('This page does not exist or you do not have authorized access.')).to be(false)
+    expect(@driver.find_elements(:css, '#feed-item-page').empty?).to be(false)
+  end
+
+  it 'should load route :team/feed/invitation', bin3: true do
+    data = api_create_feed_invitation
+    user = data[:user2]
+    team = data[:team]
+
+    # log in with user 2
+    @driver.navigate.to "#{api_path}session?email=#{user.email}"
+    sleep 2
+
+    @driver.navigate.to "#{@config['self_url']}/#{team.slug}/feed/#{data[:feed_invitation].feed['id']}/invitation"
+    wait_for_selector('.feed-invitation-container-card')
+    expect(@driver.page_source.include?('This page does not exist or you do not have authorized access.')).to be(false)
```

Comment:
Hi @danielevalverde! I noticed you're testing that the "not found" page isn't displayed. Please also add to another assertion to each test for negative "crash" page content as well: "An unexpected error happened"